### PR TITLE
Help update

### PR DESCRIPTION
**Pull request by TitasKu**

### CODE REVIEW

**Comment by TitasKu** [comment]
Merged all pull requests into this one.

**Comment by TitasKu** [comment]
Hi Utkarsh,

Thanks for the feedback, corrected the sentence.

File: docs/experiment-with-the-sample-app.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## 1. Web App
 
-You will first need to register for a free account on [Intempt Platform](https://app.intempt.com/login "Intempt Platform").
+You will first need to register for a free account on the [Intempt Platform](https://app.intempt.com/login "Intempt Platform").
```
Comment (sidchaudhary) [inline docs/experiment-with-the-sample-app.md:5]: "The" is not correct in this context. "Intempt Platform" is a noun. Build on "Intempt" is correct. Build on the Intempt is wrong.

File: docs/five-steps-to-success.md [hunk 2]
```diff
@@ -19,19 +19,19 @@ Skills Required | Language (if applicable) | Approx Duration
...
-Back-end web developer (server side SDKs) | Java, JavaScript, PHP, or Ruby | 1-2 weeks
+Back-end web developer (server-side SDKs) | Java, JavaScript, PHP, or Ruby | 1-2 weeks
 iOS developer (Wallet SDK) | Swift or React Native | 1-2 weeks
 
 We are forever making improvements to our APIs, SDKs, and documentation and expect these times to reduce over time.
 
 ### a. Sign-up on Intempt Platform ###
 
-The first thing you need to do to get started with Intempt Platform is to [sign-up for an account](https://app.intempt.com/login "Intempt SignUp"). You will have to activate your account using the activation link sent to you upon registration.
+The first thing you need to do to get started with the Intempt Platform is to [sign-up for an account](https://app.intempt.com/login "Intempt SignUp"). You will have to activate your account using the activation link sent to you upon registration.
```
Comment (sidchaudhary) [inline docs/five-steps-to-success.md:29]: No "the". Please remove everywhere.

File: docs/sample-use-case.md [hunk 3]
```diff
@@ -4,25 +4,25 @@
...
 
 ![img](intempt_img_1.png)
 
-After searching and booking a reservation the user is asked to download the App to complete their reservation since the App is central to their restaurant experience. 
+After searching and booking a reservation, the User is asked to download the App to complete their reservation since the App is central to their restaurant experience.
 
 ## 2. Workflow:  Download App and Login
 
-After making a reservation, the User is asked to download the app and login to see what their experience at the restaurant will be like.
+After making a reservation, the User is asked to download the App and login. Now the User can check out what type of experience he can expect at the restaurant.
```
Comment (sidchaudhary) [inline docs/sample-use-case.md:21]: Do not use "he" - it is considered sexist.

File: docs/sample-use-case.md [hunk 4]
```diff
@@ -34,15 +34,15 @@ The App introduces the User to the concept of earning tokens (from your loyalty
 
 ![img](image9.png)
 
-Users are encouraged to explore the menu to order while in the restaurant to order and earn tokens in the process.
+While in the restaurant, the User is encouraged to explore the menu to order and earn tokens in the process.
 
 ![img](image6.png)
 
-## 4. Workflow: Finally they review their options and place their order
+## 4. Workflow: Finally, User reviews the options and place the order
```
Comment (sidchaudhary) [inline docs/sample-use-case.md:41]: This is gramatically incoorect. The previous sentence was gramtically correct. Why the change?

**Comment by TitasKu** [comment]
Thanks for the input, corrected the issues.

**Comment by sidchaudhary** [comment]
@TitasKu what's the plan for this PR?

**Comment by TitasKu** [comment]
It's all ready. We should merge it before we make additional corrections in the help site.
